keras/utils/timeseries_dataset.py

```diff
@@ -69,6 +67,12 @@ def timeseries_dataset_from_array(
       shuffle: Whether to shuffle output samples,
         or instead draw them in chronological order.
       seed: Optional int; random seed for shuffling.
+      drop_remainder: Optional bool; representing whether the last windows
+        should be dropped if their size is smaller than size.
+      many_to_many: Optional bool; whether the target
```

Comment:
This seems like a fairly confusing API, please remove.

Reply:
Ok!